docs/content/guides/developer/sui-101/using-events.mdx

```diff
@@ -207,7 +207,13 @@ async fn main() -> Result<()> {
 
 ## Filtering event queries
 
-To filter the events returned from your queries, use the following data structures. Currently, you cannot combine filters.
+To filter the events returned from your queries, use the following data structures.
+
+:::caution
+
+This set of filters only applies to **Event querying APIs**. It differs from the filters offered for the subscriptions API (see following section), and in particular it **does not** support combinations, like `"All": [...]`, `"Any": [...]`, `"And": [_, _]`, `"Or": [_, _]`, or `"Not": _`. 
```

Comment:
```suggestion
This set of filters applies only to event querying APIs. It differs from the filters offered for the subscriptions API (see following section). In particular, it does not support combinations like `"All": [...]`, `"Any": [...]`, `"And": [_, _]`, `"Or": [_, _]`, and `"Not": _`. 
```
This is just a small nit pile. Especially that and/or at the end there. I don't think bolding is ever really needed except for UI elements (I think it's in the style guide).